src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -1466,7 +1467,7 @@ export class SearchView extends ViewletPanel {
 
 			dom.append(messageEl, $('p', undefined, resultMsg));
 			this.reLayout();
-		} else if (!msgWasHidden) {
+		} else if (msgWasHidden) {
```

Comment:
Is this right? I don't understand the purpose of the original code exactly

Reply:
Needed that for a bit, then just now went back and reverted it and the behavior is the same. 🤷‍♀️